app/views/team_mailer/user_added_to_team.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1><%= t('mailer.hello', firstname: @user.first_name) %></h1>
+<p><%= t('mailer.team_mailer.user_added_to_team.added_to_team_info', team_name: @team.name) %></p>
```

Comment:
Thought about a link to team show page here as well as in `team_registered_to_event.html.erb`?

Reply:
Good idea – done! 🙂